tests/State/HubTest.php

```diff
@@ -292,4 +292,13 @@ public function testCaptureEvent(): void
 
         $this->assertEquals('2b867534eead412cbdb882fd5d441690', $hub->captureEvent(['message' => 'test']));
     }
+
+    private static function getScope($hub): Scope
```

Comment:
Why `static`?

Also:
```suggestion
    private static function getScope(Hub $hub): Scope
```

Reply:
Even better, `HubInterface`